packages/cuid/cuid.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+authors: "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+homepage: "https://github.com/marcoonroad/ocaml-cuid"
+bug-reports: "https://github.com/marcoonroad/ocaml-cuid/issues"
+license: "MIT"
+dev-repo: "git://github.com/marcoonroad/ocaml-cuid.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+install: [make "install"]
+build-test: [make "test"]
```

Comment:
With dune/jbuilder this is usually just: `build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]`